### DISCUSSION TITLE
Is there any plans to have a Prism Maui Release candidate

### DISCUSSION THREAD

**Discussion by developer9969** [PrismLibrary/Prism#2544] [General] [answered]
Hi, 
We are planning to start porting our apps to MAUI when it goes into "RC" and we use prism , is there any plans to have a "Prism Maui release candidate"

Many thanks

**Comment by dansiegel** [comment, marked as answer]
I will get to it when i have the time, it's been slow going as the tooling hasn't quite been there to really build/debug. If you want to make the switch to Maui when it's "RC" that's your choice but you should expect a painful transition as the community needs time for things to stabilize and release updated packages.

**Reply by developer9969** [reply]
Thanks for your reply. Totally get it. We will wait as we have no option
